example/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
     sdk: flutter
 
   http: ^0.13.0
-  oktoast: ^3.2.0
+  oktoast: ^3.1.0
```

Comment:
Is this intended?

Reply:
My flutter 3.x.x sdk have some error, oktoast 3.2 cannot run on my device ,I think oktoast version in example maybe to 'any'.